### ISSUE TITLE
Add mention of adblockers/script blockers if a connection to server can't be made 

### ISSUE THREAD

**Issue by Arielle-Shander** [GDQuest/learn-gdscript#57]
**Describe the bug**
When doing lesson 2's "Fix Your First Error" in Firefox, clicking Run outputs the message: "ERROR The script has an error, but the language server didn't catch it. Are you connected?" However, the lesson works normally in Chrome.

**To Reproduce**
Steps to reproduce the bug:
1. Go to the 'Fix Your First Error' practice in Firefox
2. Click on 'Run'
3. Read Output section

**Expected behavior**
A message explaining that the "return" keyword needs to be put in the code.

**Screenshots**
![OutputError](https://user-images.githubusercontent.com/10843835/147700681-8d895b51-3020-4864-8d42-5ee73aa4f89b.PNG)

**Information about your device (please complete the following information):**
 - Operating System: Windows 10
 - Browser: Firefox

**Additional context**
Happens in Firefox but works normally in Chrome.

**Comment by Xananax** [comment]
I can't reproduce it in Firefox. It's likely this happened to you while the server was updating and therefore offline for a spell, but I can't be sure. Is it still happening?

If yes, may I ask you to:

1. Open `☰` > `More Tools` > `Developer Tools`
2. Press the <kbd>Run</kbd> button again in the app.
3. Check the "Network" tab and see if the request resolves correctly (like on the screenshot below)

![image](https://user-images.githubusercontent.com/836082/147756779-7930f167-6aeb-4a6d-a35e-df2b8ce3deec.png)

If there are no network errors, can you copy the contents of the "Console" tab?

Thank you!

**Comment by Arielle-Shander** [comment]
Hey, Xananax. Thanks for your response. 

Using the Network tab helped me troubleshoot the issue. It turns out that my NoScript extension was blocking that particular feature. I made an exception in NoScript for the gdquest.github.io page when I first opened it, but I didn't realize that I didn't let gdquest.com through. So it turned out to be human error. Sorry about that! I'll let you know if I find anything else!

![NoScript](https://user-images.githubusercontent.com/10843835/147764470-ae6717aa-eaa3-4e27-abab-eb4ec8c965d8.png)
